include/class.queue.php

```diff
@@ -1293,7 +1293,8 @@ static function create($vars=false) {
 
         $queue = new static($vars);
         $queue->created = SqlFunction::NOW();
-        $queue->setFlag(self::FLAG_QUEUE);
+        if (isset($vars['flags']) && $vars['flags'] != 0)
```

Comment:
I believe this is incorrect  - `FLAG_QUEUE` shouldn't be set when `$vars['flags']` is set.